scripts/build.d/numpy

```diff
@@ -4,10 +4,10 @@ set -e
 
 pkgname=numpy
 pkgbranch=${VERSION:-main}
-pkgfull=${pkgname}-${pkgbranch}
+pkgfull=${pkgname}
 
 # unpack (clone / pull)
-syncgit https://github.com/numpy ${pkgname} ${pkgbranch}
+syncgit https://github.com/numpy ${pkgname} ${pkgbranch} ${pkgfull}
```

Comment:
Use `numpy` as the directory name so that when switching the branch, we do not need to clone the repository again to a new directory.  It saves a lot of bandwidth.

In the future, we may add an option for different branches to use different directories.